xbmc/storage/android/AndroidStorageProvider.cpp

```diff
@@ -29,11 +29,29 @@
 #include "filesystem/File.h"
 #include "filesystem/Directory.h"
 
+#include "Util.h"
 #include "utils/log.h"
 #include "utils/RegExp.h"
 #include "utils/StringUtils.h"
 #include "utils/URIUtils.h"
 
+static const char * typeWL[] = { "vfat", "exfat", "sdcardfs", "fuse", "ntfs", "fat32", "ext3", "ext4", "esdfs" };
+static const char * mountWL[] = { "/mnt", "/Removable", "/storage" };
+static const char * mountBL[] = {
+  "/mnt/secure",
+  "/mnt/shell",
+  "/mnt/asec",
+  "/mnt/obb",
+  "/mnt/media_rw/extSdCard",
```

Comment:
Yes otg is mounted through fuse as well (mountpoint in media_rw and then fuse service creates the /storage/usbdisk)